cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -354,16 +348,12 @@ func TestGetAPIServerCommand(t *testing.T) {
 			},
 		},
 		{
-			name: "auditing is enabled with a custom log max age of 0",
+			name: "HA is enabled",
```

Comment:
This is interesting and I wonder if the correct answer is to remove this test? Originally yesterday it was `HA and auditing are enabled` Someone else removed the HA part and then I removed the auditing. If I don't remove the test, what should the tests name be? "Pretty standard bland test"? :)

Reply:
+1 for removal.

Reply:
removing.